### ISSUE TITLE
LiveAnnouncer's announce is not working on the first trigger for NVDA and JAWS

### ISSUE THREAD

**Issue by uyanga-gb** [adobe/react-spectrum#5191]
### Provide a general summary of the issue here

When using the announce method of [LiveAnnouncer](https://github.com/adobe/react-spectrum/blob/main/packages/%40react-aria/live-announcer/src/LiveAnnouncer.tsx) in @react-aria for the very first time, the message is not being announced as expected. However, subsequent triggers seem to work as expected. This issue is specific to NVDA and JAWS; it functions correctly with VoiceOver.

### 🤔 Expected Behavior?

The announce method should work consistently across all triggers, starting from the first one.

### 😯 Current Behavior

The message is not announced during the first trigger of the announce method, but it works on subsequent triggers.

### 💁 Possible Solution

- There's a potential race condition between the Assistive Technology and the creation of the live area.
- A suggested solution is to create the live region during the load time of the LiveAnnouncer module instead of waiting until the first invocation of announce.

### 🖥️ Steps to Reproduce

Please check the sandbox https://codesandbox.io/s/silly-hill-99v7q7?file=/src/App.js

### Version

3.25.1

### What browsers are you seeing the problem on?

Microsoft Edge

### What operating system are you using?

Windows

### 🧢 Your Company/Team

Adobe/Alert One

**Comment by yihuiliao** [comment]
Reproduced using AssistivLabs. Also found in FireFox with NVDA and Chrome with Jaws (though this is not extensive, just the ones I tested).
